src/utils/api.ts

```diff
@@ -6,10 +6,21 @@ import { matchPath } from 'react-router'
 import { paths } from './apiSchema'
 import authStore from './authStore'
 
+const apiUrl = process.env.REACT_APP_API_URL
+const apiKey = process.env.REACT_APP_API_KEY
+
+if (!apiUrl) {
+  throw new Error('Missing environment variable: REACT_APP_API_URL')
+}
+
+if (!apiKey) {
+  throw new Error('Missing environment variable: REACT_APP_API_KEY')
+}
```

Comment:
I moved the two checks into a for loop. In the future, I suppose this could be moved up to validate the presence and completeness of the .env file more generally.